src/content/developers/docs/smart-contracts/security/index.md

```diff
@@ -504,6 +504,8 @@ If you plan on querying an on-chain oracle for asset prices, consider using one
 
 - **[HashEx](https://hashex.org/)** - _HashEx focuses on blockchain and smart contract auditing to ensure the security of cryptocurrencies, providing services such as smart contract development, penetration testing, blockchain consulting._
 
+- **[Inference](https://inference.ag/)** - _Security auditing company, specialized in smart contract auditing for EVM-based blockchains. Thanks to its expert auditors they identify potential issues and suggest actionable solutions to fix them before deployment._
+
```

Comment:
Per [guidelines](https://ethereum.org/en/contributing/adding-products/#product-ordering) we should list new products at the end.